src/include/block.h

```diff
@@ -211,6 +211,13 @@ struct __wt_bm {
     size_t maplen;
     void *mapped_cookie;
 
+    /* For trees (e.g., tiered tables) that are backed by multiple files or objects */
+    bool is_multi_handle;
+    WT_RWLOCK handle_table_lock;   /* Lock for block handle table */
+    WT_BLOCK **handle_table;       /* Table of block handles */
+    size_t handle_table_allocated; /* Size of handle table */
+    u_int handle_table_next;       /* Next open slot */
```

Comment:
Yes, table is especially overloaded in a database product... 

I just used that term because it's what we used in the design write-up. I've renamed it to the block handle `array` in the code. 

In the future this may change to a hash table or some other structure. But we can always rename it again if that seems appropriate.